src/lib/homestore_backend/resync_pg_shard.fbs

```diff
@@ -0,0 +1,25 @@
+namespace homeobject;
+
+table Member {
+   uuid : [ubyte];
+   name : [ubyte];
+}
+
+table PGInfoEntry {
```

Comment:
Should we reuse PG.descriptor and shard from nuobject proto ? Also the root table if it has a type whether its a blob or shardpg info ? Or we can assume that if obj_id = 0 its PGShardInfo and others will be BlobDataBatch

Reply:
yes, that is the design we discussed by reading the obj_id as you mentioned above.